doc/html/python.html

```diff
@@ -836,9 +836,9 @@ <H3>
 	Throws an exception if there is no user interface.</TD>
     </TR>
     <TR>
-      <TD><CODE>askChoice</CODE>s</TD>
+      <TD><CODE>askChoices</CODE></TD>
       <TD><CODE>(title,question,answers,<BR>
-	[def])</CODE></TD>
+	[default,multiple])</CODE></TD>
```

Comment:
Need to document the keywords for the optional arguments somewhere here. (No need to do that for the mandatory ones.)

Reply:
How would you do that? Put equals signs after them? Then it's not clear they're also possibly positional...

Reply:
You could follow the pattern similar to `addContextualSubtable()` and then note in the description that those parameters can also be passed by position if desired.

Reply:
Sure, why not :)

https://github.com/fontforge/fontforge/pull/4143/commits/7d493b04a0905c76ec55c304aa2ff80015c3490c